ServantPersistent/src/Models.hs

```diff
@@ -29,8 +29,7 @@ DBTenant json
     status TenantStatus
     createdAt UTCTime
     updatedAt UTCTime
-    UniqueTenant name
-    UinqueBackofficeDomain backofficeDomain
+    UniqueBackofficeDomain backofficeDomain
```

Comment:
There is nothing wrong with Persistent in this case. The uniqueness constraint can be called whatever you like.